gatsby-config.js

```diff
@@ -101,5 +101,14 @@ module.exports = {
       },
     },
     'gatsby-plugin-react-helmet',
+    {
+      resolve: 'gatsby-plugin-sentry',
+      options: {
+        dsn: process.env.SENTRY_DSN,
+        environment: process.env.NODE_ENV,
+        enabled: (() =>
+          ['production', 'staging'].indexOf(process.env.NODE_ENV) !== -1)(),
```

Comment:
Should have confirmed this- But assuming we've set our env vars in Netlify as `production` and `staging`?

Reply:
Yeah the env names are all default so production is what we've got on Netlify!